web/core/models.py

```diff
@@ -83,3 +90,28 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+def get_file_name_storage(instance, filename):
+    return os.path.join(str(instance.kid.id), filename)
+
+
+class KidImage(Model):
+    kid = ForeignKey(Kid, on_delete=CASCADE, verbose_name="criança")
+    image = FileField(
+        upload_to=get_file_name_storage,
+        validators=[
+            FileExtensionValidator(allowed_extensions=["jpg", "jpeg", "png", "gif"])
+        ],
+    )
+
+    @property
+    def image_render(self):
+        return mark_safe('<img src="%s" width="150" height="150" />' % (self.image.url))
```

Comment:
Já que estamos no Python 3, vamos usar _f-strings_?

```suggestion
        return mark_safe(f'<img src="{self.image.url}" width="150" height="150" />')
```